configurations/test/env.yml.tmp

```diff
@@ -0,0 +1,25 @@
+AUTH0_API_CLIENT: test-auth0-client-id
+AUTH0_API_SECRET: test-auth0-secret
+AUTH0_DOMAIN: test.auth0.com
```

Comment:
I think it is. Binh has asked me to include it in other PR reviews.

Reply:
@binh-dam-ibigroup, can you chime in as to why you wanted this file included? I'm having trouble finding the original comment.

Reply:
@landonreed, this file is no longer needed. I thought originally that this file was needed because of secrets, however @br648 mentioned the actual `test/env.yml` is needed to run CI tests.

Reply:
Have removed.